<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat</class>
 <widget class="QWidget" name="chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>438</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>438</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>438</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>330</width>
     <height>441</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>330</width>
     <height>441</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>330</width>
     <height>441</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/sky.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="historyTextBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>311</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Comic Sans MS&quot;;
background-color: rgba(255, 255, 255,0.5);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="chatLineEdit">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>327</y>
     <width>240</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>31</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;Comic Sans MS&quot;;
background-color: rgba(255, 255, 255,0.5);</string>
   </property>
  </widget>
  <widget class="QToolButton" name="sendToolButton">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>327</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>71</width>
     <height>30</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="0">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="PlaceholderText">
       <brush brushstyle="NoBrush">
        <color alpha="128">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="styleSheet">
    <string notr="true">#toolButton:hover
{
	border-style:none;
	font: 12pt &quot;华文隶书&quot;;
color: rgba(0, 0, 0, 1);
}
#toolButton:!hover
{
	border-style:none;
	font: 12pt &quot;华文隶书&quot;;
	color: rgba(0, 0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/Send.512.png</normaloff>:/icon/Send.512.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
   <property name="arrowType">
    <enum>Qt::NoArrow</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="closeToolButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/Cancel.512.png);</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="showLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px groove gray;border-radius:
7px;padding:2px 4px;
font: 9pt &quot;Candara&quot;;</string>
   </property>
   <property name="text">
    <string>显示用户名</string>
   </property>
  </widget>
  <widget class="QToolButton" name="receiveFileToolButton">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>381</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>44</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>71</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#toolButton_3:hover
{
	border-style:none;
	font: 12pt &quot;华文隶书&quot;;
color: rgba(0, 0, 0, 1);
}
#toolButton_3:!hover
{
	border-style:none;
	font: 12pt &quot;华文隶书&quot;;
	color: rgba(0, 0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>接收文件</string>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/Downloads-Folder.512.png</normaloff>:/icon/Downloads-Folder.512.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="sendFileToolButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>381</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>71</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>71</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#toolButton_4:hover
{
	border-style:none;
	font: 12pt &quot;华文隶书&quot;;
color: rgba(0, 0, 0, 1);
}
#toolButton_4:!hover
{
	border-style:none;
	font: 12pt &quot;华文隶书&quot;;
	color: rgba(0, 0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>发送文件</string>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/icon/sendfile icon.512.png</normaloff>:/icon/sendfile icon.512.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <zorder>backgroundLabel</zorder>
  <zorder>chatLineEdit</zorder>
  <zorder>sendToolButton</zorder>
  <zorder>showLabel</zorder>
  <zorder>historyTextBrowser</zorder>
  <zorder>closeToolButton</zorder>
  <zorder>receiveFileToolButton</zorder>
  <zorder>sendFileToolButton</zorder>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
